lesson05/assignment/index.html

```diff
@@ -1,14 +1,59 @@
+<!DOCTYPE html>
 <html lang="en">
-    <head>
-        <meta charset="UTF-8" />
-        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-        <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-        <link rel="stylesheet" href="css/style.css" />
-        <title>Assignment 05</title>
-    </head>
-    <body>
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Assignment 05 - CSS Grid</title>
+</head>
+
+<body>
+    <div class="container">
+        <header>
+            <h1 class="heading">Heading</h1>
+            <nav class="nav">
```

Comment:
For the BEM requirement, you can just update the nav component and its children to utilize some sort of BEM classes.

Reply:
Updated!